docs/getting-started-guides/aws.md

```diff
@@ -39,18 +32,105 @@ export KUBERNETES_MASTER=https://<ip-address>
 
 Also setup your path to point to the released binaries:
 ```
-export PATH=$PATH:$PWD:/cluster
+export PATH=$PATH:$PWD:/kubernetes/cluster
 ```
 
-### Running examples
+If you run into trouble please see the section on [troubleshooting](https://github.com/brendandburns/kubernetes/blob/docs/docs/getting-started-guides/gce.md#troubleshooting), or come ask questions on IRC at #google-containers on freenode.
```

Comment:
The troubleshooting link is mostly about troubleshooting GCE env. Probably not relevant here. Also, the link points to github,com/brendanburns/kuberenets/.. instead of head.